spec/presenters/alarm_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
```

Comment:
I highly recommend moving your test files to the same root path: `spec` or `tests` to facilitate visualization. 
Also, try to avoid hardcoding values to test different values and catch additional error scenarios. I assume your factory file has some hardcoded values.